### ISSUE TITLE
Improve explanation on Python executable for Windows users

### ISSUE THREAD

**Issue by Julietadeboye** [wagtail/wagtail#9336]
### Pertinent section of the Wagtail docs

[Getting started tutorial](https://docs.wagtail.org/en/stable/getting_started/tutorial.html?_ga=2.209345426.1239746541.1651239819-1126786067.1648068945#install-and-run-wagtail)

### Details

I noticed an error in the getting started tutorial which might be a blocker for Windows users.
Python3.exe is not defined in windows so each time you want to run a command that includes python3 you will get the error "'python3' is not recognized as an internal or external command, operable program or batch file"
The alternative for this on windows is py -3
I am working on this edit and will sub
![Screenshot (421)](https://user-images.githubusercontent.com/62646559/195580205-3cbeb5a1-8733-41f1-8b97-04e1c3e6a9e5.png)
mit a pull request shortly .

**Comment by Julietadeboye** [comment]
@thibaudcolas  @lb-

**Comment by dchidindu5** [comment]
Thanks for this, I was about raising an issue on this and lucky enough I saw it here.

**Comment by thibaudcolas** [comment]
Hi @Julietadeboye, thanks for opening this here! No need to @-mention mentors unless there’s something urgent for us to do, we regularly review all issues.

Also note as a point of etiquette – if you open a new issue, it’s usually a good idea to wait a bit for feedback before also opening a pull request. This is so you don’t spend time working on something we might disagree with, and also because if we’re not careful there is a risk the conversation might become split between the issue and the pull request.

Back to your change – this looks promising to me but I’ve got to say I’ve never seen this `py -3` command over all the years I’ve used Windows for on and off. So I think we need to know:

- Which version of Windows we’re talking about here
- Which shell (Powershell, Git Bash, WSL, other)
- How you installed Python

Additionally, it’s worth mentioning very well known tutorials like https://tutorial.djangogirls.org/en/python_installation/ use `python3`. Could you share any examples of Python or Django tutorials using `py -3` ?

**Comment by lb-** [comment]
Interestingly, Django seems to use `py` (not  `py -3`).

I have only seen one example of this and have not dug deep into the Django issue behind this.

<img width="960" alt="Screen Shot 2022-10-15 at 8 51 21 am" src="https://user-images.githubusercontent.com/1396140/195954855-6f93ec92-3be5-4748-b9ff-83a6018e62dc.png">

But again, I have not seen the usage of `py -3` come up in any Windows getting started feedback over the last few years so not sure if that's a new thing.

@Julietadeboye could you please add more detail, as per Thibaud's comment above, try to find the Django issue https://code.djangoproject.com/ OR the Django PR that related to this change above and maybe that will have further insights as to why they went in this direction.

https://github.com/django/django

**Comment by Julietadeboye** [comment]
Hi Thibaud and LB.

Thanks for your feedback. Will do better in my next contribution.

- I use Windows 10, but this is an issue experienced by versions. 
- I used Command prompt for my installation
- I followed the [tutorial](https://docs.wagtail.org/en/stable/getting_started/tutorial.html#install-dependencies) on how to install the required version of Python [here](https://www.python.org/downloads/)

After installation, using the command `python3` on windows gives this error "'python3' is not recognized as an internal or external command, operable program or batch file" as shown below:

![Screenshot (426)](https://user-images.githubusercontent.com/62646559/195977264-86131291-d0cf-4480-a7b4-3c1ff86a09a4.png)

I tried to debug this by checking [StackOverflow](https://stackoverflow.com/questions/39910730/python3-is-not-recognized-as-an-internal-or-external-command-operable-program) for answers 

As you mentioned, a more known alternative is to use `py` but if you have more than one version of python,
you can specify which version you want to launch by using `py -3` or `py -2`. I suggested the use of `py -3` because the wagtail project specifically supports this version. 

I am yet to find a django tutorial or documentation that directly speaks to this. I'll keep searching and will share here if I find any.

**Comment by thibaudcolas** [comment]
Based on https://docs.python.org/3/using/windows.html, it seems `py` is the "Python Launcher for Windows", which will automatically start Python with the latest version possible – and allow you to use other versions if you want to. In our case, we can definitely use whichever version of Python is the latest, so we don’t need the `-3`. And DjangoGirls doesn’t use it because their tutorial has users click "Add Python 3.x to PATH".

So – I think we could perhaps change our Python installation section at the top of the tutorial to call out the different commands. Something like:

    To check whether you have an appropriate version of Python 3:

    ```sh
    python --version
    # Or sometimes:
    python --version
    # Or on Windows:
    py --version
    ```

    Of all three options, use the one that works for you throughout this tutorial. If this does not return a version number or returns a version lower than 3.7 […]

What do you think?

**Comment by Julietadeboye** [comment]
Hi  Thibaud, `py --version` did show the latest version. Thanks,
Did you mean to write
```sh
**python3 --version**
# Or sometimes:
python --version
# Or on Windows:
py --version
```
The same command was repeated.

**Comment by thibaudcolas** [comment]
Yes I did :) Though I think `python` seems more universal than `python3`?

**Comment by Julietadeboye** [comment]
It is, and since there will be newer versions in the further, this part of the documentation will not need to be revisited.

**Comment by Julietadeboye** [comment]
Hey Thibaud, should I go ahead to make the changes?

**Comment by thibaudcolas** [comment]
@Julietadeboye if you think it’s a good idea then yes! You can update your existing pull request, https://github.com/wagtail/wagtail/pull/9338

**Comment by Julietadeboye** [comment]
Alright, will do. Thanks!

**Comment by dchidindu5** [comment]
In Windows, python3 doesn't work
Preferably follow the instructions from the mentor.

**Comment by Julietadeboye** [comment]
@dchidindu5 not sure what you're referencing here but yes, `python3` doesn't work in Windows

**Comment by thibaudcolas** [comment]
Fixed in #9338!
